### ISSUE TITLE
Configure default "Tweak available layout" set & default panels icons

### ISSUE THREAD

**Issue by Ateliers-Modernes** [material-shell/material-shell#628]
Hey,

A way to save "tweak available layout" as default for new "panel" creations.

Thx

**Comment by Oleg-dM** [comment]
Yeah that would be awsome, I only use simple horizontal and need to add it every time I create a new workspace which happens .. a lot. Would save a lot of time for many people I'm sure.
Thanks for the great work anyway!
